instance/github.py

```diff
@@ -164,6 +191,13 @@ def extra_settings(self):
         return get_settings_from_pr_body(self.body)
 
     @property
+    def ephemeral_databases(self):
```

Comment:
Same remark about the name as in the function above. `use_ephemeral_databases()`?